ui/pages/routes/routes.component.test.js

```diff
@@ -71,7 +71,7 @@ describe('Routes Component', () => {
     mockHideNetworkDropdown.mockClear();
   });
   describe('render during send flow', () => {
-    it('should render with network change disabled while adding recipient for send flow', () => {
+    it.only('should render with network change disabled while adding recipient for send flow', () => {
```

Comment:
I think you pushed this by mistake
```suggestion
    it('should render with network change disabled while adding recipient for send flow', () => {
```

Reply:
Fixed in 55991ab71addbf1b4c013d0eafbe913870ab97b9